### DISCUSSION TITLE
ScriptReference/ParticleSystem.Stop

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#542] [Notes]
# ScriptReference/ParticleSystem.Stop

https://docs.unity3d.com/ScriptReference/ParticleSystem.Stop.html

**Comment by joethephish** [comment]
I've found that after calling `particleSystem.Stop()`, then `particles.isStopped` may still return `false`. However, it silently in the background may be shutting down all the while it still has particles alive, and only return `isStopped` when they're all dead. I'm not sure how to reliably tell when you have successfully called `Stop()` and it's definitely on the way to switching off.

**Reply by ogike** [reply]
From my quick experience, looks like isStopped will return true as long as there are alive particles.
If you want to check when the particle system has stopped emitting (from a Stop() command for example), use the isEmitting bool instead.
